README.md

```diff
@@ -30,6 +30,14 @@ However there are a couple of differences:
 * `sleuth` might support different set of instrumentations.
 * Instrumentations in `sleuth` might add different set of tags and logs to represent the same events.
 
+## Note on dependencies
+
+It's worth noting that the although OpenTracing Spring Cloud contains code for instrumenting a wealth of Spring projects, 
+it however does not pull those dependencies automatically, having dependencies as optional dependencies instead.
```

Comment:
I am very far from a native speaker but this `having dependencies as optional dependencies instead.` could be maybe rephrased.

Reply:
I am a native speaker and although it rings true to, the fact that it doesn't sound good to someone else, means that it can be improved :). And improve I shall try :P

Reply:
Then I would rather believe you! :)

Reply:
Merge on green please